src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -91,7 +97,7 @@ public CheckInProcessRecords withCheckInServicePoint(ServicePoint checkInService
       this.loan,
       this.requestQueue,
       checkInServicePoint,
-      this.highestPriorityFulfillableRequest);
+      this.highestPriorityFulfillableRequest, loggedInUserId);
```

Comment:
Similar to other comments, please either format the parameters up to end of a line or as a parameter per line, rather than a mixture.

Reply:
Done.